tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -73,6 +74,8 @@ CO_PROCESSOR :=
 # include directories from one source.
 INCLUDES := \
 -I. \
+-I$(TENSORFLOW_ROOT) \
+-Igoogle \
```

Comment:
let's change to be `-I$(EXTERNAL_INCLUDE_DIR)`

Reply:
Done. Changed to EXTERNAL_DIR